apps/edxapp/vars/all/main.yml

```diff
@@ -112,7 +71,7 @@ edxapp_nginx_cms_admin_ip_withelist: []
 edxapp_nginx_cms_bypass_htaccess_ip_whitelist: []
 edxapp_nginx_lms_admin_ip_withelist: []
 edxapp_nginx_lms_bypass_htaccess_ip_whitelist: []
-edxapp_nginx_cms_static_cache_expires: "1M"
+edxapp_nginx_cms_static_cache_expires: "1d"
```

Comment:
Actually I should include this in a separate commit and explain that Open edX does not version static files that are called via `require` (they may change when we upgrade the base open edx image of one of our site). So we should not set cache expiries too long.

Reply:
done